### ISSUE TITLE
Question: is ESX alrady implemented?

### ISSUE THREAD

**Issue by justinsanjp** [ND-Framework/ND_MDT#22]
:D

**Comment by Andyyy7666** [comment]
Yes here: https://github.com/Maximus7474/ND_MDT/tree/ESX-Version

It's currently a pull request but will probably be merged in the coming days.

**Comment by justinsanjp** [comment]
Thanks!

**Comment by Andyyy7666** [comment]
@justinsanjp does it work properly?

**Comment by justinsanjp** [comment]
Hey abit late but Yeah it does work fine. But I switched to a custom script just for learning purposes.
